go-controller/pkg/node/gateway_init.go

```diff
@@ -103,7 +103,8 @@ func (n *OvnNode) initGateway(subnet *net.IPNet, nodeAnnotator kube.Annotator,
 				gatewayIntf = defaultGatewayIntf
 			}
 		}
-		prFn, err = n.initSharedGateway(subnet, gatewayNextHop, gatewayIntf, nodeAnnotator)
+		//FIXME DUAL-STACK
+		prFn, err = n.initSharedGateway(subnets[0], gatewayNextHop, gatewayIntf, nodeAnnotator)
```

Comment:
`initSharedGateway` doesn't even look at its `subnet` argument, so you can just remove it and not need a FIXME.

Reply:
Will do.